OrbitCore/BlockChain.h

```diff
@@ -111,8 +111,30 @@ class BlockChain final {
  public:
   BlockChain() : size_(0) { root_ = current_ = new Block<T, BlockSize>(nullptr); }
 
+  BlockChain(const BlockChain& other) = delete;
+
+  BlockChain& operator=(const BlockChain& other) = delete;
+
+  BlockChain(BlockChain&& other) {
+    root_ = other.root_;
+    current_ = other.current_;
+    size_ = other.size_;
+  }
+
+  BlockChain& operator=(BlockChain&& other) {
```

Comment:
s/other/that/? not sure if this is better.

Reply:
you mean the name of the variable? I'd guess "other" should be fine. Otherwise, I could use "that".